nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -34,6 +34,7 @@ def check_samplesheet(file_in,file_out):
     sample_run_dict = {}
     with open(file_in, 'r') as fin:
 
+        # TODO nf-core: update the header names for the input samplesheet
```

Comment:
```suggestion
        # TODO nf-core: Update the column names for the input samplesheet
```

Reply:
Great idea with the `TODO` :+1:  could maybe add another one asking to change/delete the script as a whole too depending on more specific pipeline inputs.

Reply:
I'll do that! Actually just created a pipeline from the template to check and trying to understand now the structure, and if I find more stuff that the user needs to update, I'll add TODOs

Reply:
Nice! Should also maybe add a link to an example samplesheet on `test-datasets` or physically add one to `assets`